src/core/core.controller.js

```diff
@@ -405,7 +408,7 @@ module.exports = function(Chart) {
 						}
 					}
 				}
-			}).call(me);
+			}.call(me);
```

Comment:
can probably remove the `.call(me)` because `me` is captured by scope now. that could be a different PR though